ecommerce-website/src/components/products/Products.css

```diff
@@ -45,94 +45,76 @@
 }
 
 /* IndividualProductPage - Start */
-body {
```

Comment:
Just wondering what are the changes to the CSS file ? Cause I thought the one on the main branch was the good one to use!

Reply:
These are the changes: 

1. Refined the display of the product's info: 
I kept the styles for productCards, but changed for IndividualProductPage because I was not proud of the way it was coded regarding the layout, which is why I added Col (column) as well as reduced the amount of className & div. 

2. Focused on the image's size and the +, - buttons design: 
- The image's size changed because it's more responsive.
- I changed the buttons' design since they were not appearing with the implemented style. 

3. Also, added comments for maintenance and better understanding

But if this doesn't work, I could re-edit the CSS file.

Reply:
Sounds like good changes, just wanted to make sure it was on purpose :)